contracts/cw20-base/src/allowances.rs

```diff
@@ -283,6 +289,10 @@ mod tests {
         query_balance(&deps, address.into()).unwrap().balance
     }
 
+    fn mock_env_info<U: Into<HumanAddr>>(sender: U, sent: &[Coin]) -> (Env, MessageInfo) {
```

Comment:
I would remove this and normalize to the style in the other contracts.
Consistency is valuable